### PR TITLE
[T10] question creation card

### PR DESCRIPTION
**Pull request by vaibhav-patil07**
1. Added option component
2. Added Question component
3. Added Error message component

### CODE REVIEW

**Review by harshithad0703 — APPROVED** [review]

File: src/components/Question/index.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                    autoSize
+                    autoFocus
+                    onChange={titleOnChangeHandler}
+                    onBlur={titleOnChangeHandler}
+                    status={titleStatus}
+                />
+            </div>
+            <div>
+                <ErrorMessage errorMessage={errorMessage} />
+            </div>
```
Comment (faraazb) [inline src/components/Question/index.jsx:72]: Can you use [conditional rendering](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) here?

File: src/components/Option/index.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                            removeOnClickHandler(event, parentKey);
+                        }}
+                    />
+                }
+            />
+            <div>
+                <ErrorMessage
+                    className="errorMessage"
+                    errorMessage={errorMessage}
+                />
```
Comment (faraazb) [inline src/components/Option/index.jsx:57]: Conditional rendering

File: src/components/ErrorMessage/index.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import "./index.css";
+
+const ErrorMessage = (props) => {
```
Comment (faraazb) [inline src/components/ErrorMessage/index.jsx:3]: Once you conditionally render the error, since it's just one h2 tag, we might not need this component.

**Review by faraazb — CHANGES_REQUESTED** [review]
Error messages can be conditionally rendered, and the ErrorMessage component can be avoided. All other components might show errors differently than an H2, so it might not be very re-usable.

**Pull request merged**